### ISSUE TITLE
Uncaught divide by zero when importing a ext4 image

### ISSUE THREAD

**Issue by marcograss** [NationalSecurityAgency/ghidra#1342]
**Describe the bug**
A clear and concise description of the bug.

While batch importing a firmware, ghidra recognizes a file as a ext4 image and try to parse it.

A divide by zero exception is thrown and not handled anywhere.

**Expected behavior**

If the file is malformed, a message should be shown and the batch importing should continue.

**Environment (please complete the following information):**
 - OS: Mac OS X 10.15.2 x86_64
 - Java Version: Oracle Corporation 11.0.5
 - Ghidra Version:
Build Date: 2019-Oct-23 1737 EDT
Ghidra Version: 9.1

**Additional context**

Stack traces:

```Adding Source Files - Uncaught Exception: java.lang.ArithmeticException: / by zero
java.lang.ArithmeticException: / by zero
	at ghidra.file.formats.ext4.Ext4FileSystem.mountFS(Ext4FileSystem.java:60)
	at ghidra.file.formats.ext4.Ext4FileSystemFactory.create(Ext4FileSystemFactory.java:38)
	at ghidra.file.formats.ext4.Ext4FileSystemFactory.create(Ext4FileSystemFactory.java:29)
	at ghidra.formats.gfilesystem.factory.FileSystemFactoryMgr.mountUsingFactory(FileSystemFactoryMgr.java:176)
	at ghidra.formats.gfilesystem.factory.FileSystemFactoryMgr.probe(FileSystemFactoryMgr.java:345)
	at ghidra.formats.gfilesystem.FileSystemService.probeFileForFilesystem(FileSystemService.java:639)
	at ghidra.formats.gfilesystem.FileSystemService.probeFileForFilesystem(FileSystemService.java:578)
	at ghidra.plugins.importer.batch.BatchInfo.processAsFS(BatchInfo.java:297)
	at ghidra.plugins.importer.batch.BatchInfo.doAddFile(BatchInfo.java:226)
	at ghidra.plugins.importer.batch.BatchInfo.processFS(BatchInfo.java:336)
	at ghidra.plugins.importer.batch.BatchInfo.processAsFS(BatchInfo.java:312)
	at ghidra.plugins.importer.batch.BatchInfo.doAddFile(BatchInfo.java:226)
	at ghidra.plugins.importer.batch.BatchInfo.processFS(BatchInfo.java:336)
	at ghidra.plugins.importer.batch.BatchInfo.doAddFile(BatchInfo.java:222)
	at ghidra.plugins.importer.batch.BatchInfo.addFile(BatchInfo.java:194)
	at ghidra.plugins.importer.batch.BatchInfo.doAddFiles(BatchInfo.java:493)
	at ghidra.plugins.importer.batch.BatchInfo$AddFilesRunnable.monitoredRun(BatchInfo.java:530)
	at ghidra.util.task.TaskBuilder$TaskBuilderTask.run(TaskBuilder.java:303)
	at ghidra.util.task.Task.monitoredRun(Task.java:126)
	at ghidra.util.task.TaskRunner.lambda$startTaskThread$1(TaskRunner.java:94)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
```Uncaught Exception! 
java.lang.NullPointerException
	at ghidra.plugins.importer.batch.BatchImportDialog.addSources(BatchImportDialog.java:424)
	at ghidra.plugins.importer.batch.BatchImportDialog.lambda$showAndImport$0(BatchImportDialog.java:75)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by dev747368** [comment]
@marcograss  please try this out (if you can build from source) and let us know if it fixes / doesn't fix the issue with your specific filesystem image.

**Comment by marcograss** [comment]
I built the master branch and the bug is fixed @dev747368 , thanks for the quick fix and I'm looking forward to the 9.2 release
